<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="init_button">
      <property name="text">
       <string>init</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="imported_graph">
      <property name="text">
       <string>raw</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cleared_graph">
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="drop_button">
      <property name="text">
       <string>drop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>227</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
